go.mod

```diff
@@ -7,6 +7,7 @@ toolchain go1.22.7
 require (
 	github.com/BurntSushi/toml v1.4.0
 	github.com/andybalholm/brotli v1.1.0
+	github.com/bmatcuk/doublestar/v4 v4.7.1
```

Comment:
mega nit: But we probably don't need to introduce this dependency to glob for a max 2-depth forge-artifacts directory.

Reply:
The idea was to be generic enough that we can add new scripts in the future that might need to be more particular about which files to include and exclude. Thoughts?

Reply:
We'll use this base for all Go-based contract check scripts going forward

Reply:
I suppose it's OK if you expect the script to read non-forge-artifacts. Otherwise, it seems overkill to try to glob directories of arbitrary depth. But that's just my loosely held opinio and not a merge blocker.

Reply:
Yeah I'd expect to extend this to other scripts that may need to glob for more complex stuff so I'll keep it for now